classes/local/store/object_client_base.php

```diff
@@ -135,20 +152,20 @@ public function proxy_range_request(\stored_file $file, $ranges) {
      * Test proxy range request.
      *
      * @param  object  $filesystem  Filesystem to be tested.
-     * @return object
+     * @return result
      */
-    public function test_range_request($filesystem) {
-        return (object)['result' => false, 'error' => ''];
+    public function test_range_request($filesystem): result {
+        return new result(result::UNKNOWN, '');
```

Comment:
Maybe these should be `N/A` if not implemented? Otherwise things that dont implement them immediately get a `UNKNOWN` which is considered very bad.

Reply:
Changed to `INFO` which seems the most fitting https://moodledev.io/docs/apis/subsystems/check#result-states-of-a-check